components/03-modules/modal/modal--header-footer.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if trigger }}
+    {{ render '@modal-trigger' modalTrigger }}
+{{/if}}
+<div role="dialog"
+     aria-hidden="true"
+     id="{{ modal.id }}"
+     data-modal="{{ modal.id }}"
+     class="modal {{ modal.class }}"
+     tabindex="-1"
+     {{{ modal.attributes }}}
+ >
+    <div role="document"
+         class="modal__content {{ modalContent.class }}"
+         {{{ modalContent.attribtues }}}
+         tabindex="0"
+     >
+        <div class="modal__header">
+            {{{ header.html }}}
```

Comment:
I'd add `if` statement here to simpler implementation in views

Reply:
why? we can put in config `html: ''` ? and if there is no `header: { html: 'some data' }` it wont' be displayed (no errors)

Reply:
After our Friday meeting I think we should change it and refactor to heading and description with `if` not with html elements.